### PR TITLE
Updating Repo to reflect current status of GA

### PR DESCRIPTION
**Pull request by jmd-stripe**
Currently:

• remove /using-webhooks directory
• rename /without-webhooks to src
• remove identity_beta API header and other legacy code
• remove use_stripe_sdk param
• remove reference to webhooks from README, and updated README

### CODE REVIEW

File: src/server/server.js [hunk 1]
```diff
@@ -66,7 +65,6 @@ app.get('/', (req, res) => {
  * Serve return_url page
  */
 app.get('/next-step', (req, res) => {
-  // TODO handle sad path cases
```
Comment (jmd-stripe) [inline src/server/server.js:69]: I wasnt sure what this meant. I deleted it so I could create a comment on it

Comment (bz-stripe) [reply src/server/server.js:69]: Don't even remember anymore what I meant by that 😅 

If it's referring to declined consent and unsupported device error codes, those are handled through the stripe.js iframe events now.

If it's not referring to that, then well... nobody seems to have complained 😆

File: README.md [hunk 2]
```diff
@@ -28,38 +28,22 @@ Copy the env.example file into a file named .env in the project root folder. For
...
-You will need a Stripe account in order to run the demo. Once you set up your account, go to the Stripe [developer dashboard](https://stripe.com/docs/development#api-keys) to find your API keys and update your .env file with the keys.
+You will need a Stripe account in order to run the demo. Additionally, you will need to setup your account to use Identity, which you can do so [here](https://dashboard.stripe.com/identity/application). Once you set up your account, go to the Stripe [developer dashboard](https://stripe.com/docs/development#api-keys) to find your API keys and update your .env file with the keys.
 
 ```
 STRIPE_SECRET_KEY=<replace-with-your-test-secret-key>
 STRIPE_PUBLISHABLE_KEY=<replace-with-your-test-publishable-key>
-STRIPE_WEBHOOK_SECRET=<replace-with-your-test-webhook-secret>
 ```
 
-`CLIENT_DIR` tells the server where to the client files are located and does not need to be modified unless you move the server files.
```
Comment (jmd-stripe) [inline README.md:39]: I didnt see CLIENT_DIR in the env.example, but saw STATIC_DIR. Is this correct?

**Review by bz-stripe — APPROVED** [review]
lgtm — I also pulled and tested locally

**Pull request merged**